Resources/Prototypes/Datasets/tips.yml

```diff
@@ -30,16 +29,17 @@
   - "Riot armor is significantly more powerful against opponents that aren't using guns compared to regular armor."
   - "As a ghost, you can use the Verb Menu to orbit around and follow any entity in game automatically."
   - "As a Traitor, you may sometimes be assigned to hunt other traitors, and in turn be hunted by others."
-  - "As a Traitor, the syndicate headset can be used to communicate with any other traitors who may have purchased it through a secure channel."
-  - "As a Traitor, never forget the power of the syndicate toolbox. For only 2 telecrystals, you can get a bunch of full set of tools to boost you in an emergency, along with the toolbox itself packing a decent punch."
-  - "As a Traitor, never underestimate the web vest. It may not provide space protection, but its cheap cost and robust protection makes it handy for trigger-happy foes."
+  - "As a Traitor, the syndicate encryption key can be used to communicate with any other traitors who may have purchased it through a secure channel."
```

Comment:
It sounds like the encryption key was "purchased through a secure channel", move the clause "through a secure channel" after "communicate"